src/main/java/baseball/game/NumberBaseBallGameMessage.java

```diff
@@ -0,0 +1,51 @@
+package baseball.game;
+
+import baseball.game.entity.NumberBaseBallGameWord;
+
+public class NumberBaseBallGameMessage {
+
+    public static void printMessage(String message) {
+        System.out.println(message);
+    }
+
+    public static void printStart() {
+        System.out.println(NumberBaseBallGameWord.START.getMessage());
+    }
+
+    public static void printRunning() {
+        System.out.print(NumberBaseBallGameWord.RUNNING.getMessage());
+    }
+
+    public static void printContinue() {
+        System.out.println(NumberBaseBallGameWord.CONTINUE.getMessage());
+    }
+
+    public static String getEndMessage() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(3).append(NumberBaseBallGameWord.STRIKE.getMessage()).append("\n");
+        sb.append(NumberBaseBallGameWord.END.getMessage());
+
+        return sb.toString();
+    }
+
+    public static boolean isEnd(String message) {
+        if (message.equals(getEndMessage())) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isReStart(String message) {
+        if (message.equals(NumberBaseBallGameWord.NEW_GAME.getMessage())) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    public static boolean isReStart(String message) {
        return message.equals(NumberBaseBallGameWord.NEW_GAME.getMessage();
    }
```

이 부분은 이렇게 작성해도 그대로 가독성을 유지할 수 있을 것 같아요!

Reply:
오! 좋은 방법이네요!
감사합니당